doc/howto/storage_resize.md

```diff
@@ -0,0 +1,66 @@
+---
+discourse: 1333
+---
+
+# How to resize storage
+
+If you need more storage, you can increase the size of your storage pool or your storage volume.
+In some cases, it is also possible to reduce the size of a storage volume.
+
+## Grow a storage pool
+
+To increase the size of a storage pool, follow these general steps:
+
+1. Grow the size of the storage on disk.
+1. Let the file system know of the size change.
+1. Update the pool size configuration in LXD.
+
+See the specific commands for different storage drivers below.
+
+````{tabs}
+
+```{group-tab} Btrfs
+
+Enter the following commands to grow a loop-backed Btrfs pool by 5 G:
```

Comment:
"5 Gigabytes" maybe here? As "G" is not a unit LXD uses (although it is for the underlying Btrfs commands), but this part could use full words instead.

Reply:
Yeah, the `5G` probably comes from what `truncate` uses. But indeed, not a unit we use in LXD, so should either be 5Gb, 5GiB, 5 Gigabytes or 5 Gigibytes

Reply:
Well, that is exactly why I left the "G", since I had no idea how to resolve it. :grin:

Reply:
So I think leave the `G` on the actual command (as its needed by BTRFS command) but say "Gigabytes" on the description of the command so its human readable.